run_redash_locally.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+#Provision VM
+vagrant up
+
+#start server and background workers
+vagrant ssh -c "cd /opt/redash/current; bin/run honcho start -f Procfile.dev" &
```

Comment:
Let's have a single script to control vagrant and place it in `bin/` (we can call it `vagrant_ctl`) and use subcommands:

```bash
#!/bin/bash
set -e

help() {
  echo "Usage: "
  echo "`basename "$0"` {start, test}"
}

case "$1" in
  run)
    vagrant up
    vagrant ssh -c "cd /opt/redash/current; bin/run honcho start -f Procfile.dev"
    ;;
  test)
    vagrant up
    vagrant ssh -c "cd /opt/redash/current; make test"
    ;;
  *)
    help
    ;;
esac
```